test/unit/multipart-test.ts

```diff
@@ -1,18 +1,17 @@
 import { assert } from 'chai';
 import { setup } from 'f-mocha';
-import { run, wait } from 'f-promise';
 import { bufferReader, bufferWriter, multipartFormatter, multipartParser } from '../..';
 import { IncomingHttpHeaders } from 'http';
 setup();
 
-const { equal, ok, strictEqual, deepEqual } = assert;
+const { equal, ok, strictEqual } = assert;
 
-const boundary = '-- my boundary --';
+const boundary = '------------myBoundary';
```

Comment:
Why did you have to change the boundary?

Reply:
The format seems to be more standard like that...

The double dash at the end is used normally to specify the end of the last content boundary